api/node/typescript/index.ts

```diff
@@ -918,6 +918,30 @@ export namespace private_api {
     }
 }
 
+import * as process from "node:process";
+import * as nodePath from "node:path";
+
+/**
+ * Initialize translations.
+ *
+ * Call this with the path where translations are located. This function internally calls the [bindtextdomain](https://man7.org/linux/man-pages/man3/bindtextdomain.3.html) function from gettext.
+ *
+ * Translations are expected to be found at <path>/<locale>/LC_MESSAGES/<domain>.mo, where path is the directory passed as an argument to this function, locale is a locale name (e.g., en, en_GB, fr), and domain is the package name.
+ *
+ * @param domain defines the domain name e.g. name of the package.
+ * @param path specifies the directory in which gettext should search for translations.
+ *
+ * For example, assuming this is in a crate called example and the default locale is configured to be French, it will load translations at runtime from ``/path/to/example/translations/fr/LC_MESSAGES/example.mo`.
+ *
+ * ```js
+ * import * as slint from "slint-ui";
+ * slint.initTranslations("package-name", "/translations/")
```

Comment:
I suggest to change this to something that can be re-used, i.e. something that users can copy & paste into their app and use with minimal adjustments, if any.